pkg/apis/core/types.go

```diff
@@ -2634,6 +2634,15 @@ type PodSecurityContext struct {
 	// +k8s:conversion-gen=false
 	// +optional
 	HostIPC bool
+	// Share a single process namespace between all of the containers in a pod.
+	// When this is set containers will be able to view and signal processes from other containers
+	// in the same pod, and the first process in each container will not be assigned PID 1.
+	// HostPID and ShareProcessNamespace cannot both be set.
+	// Optional: Default to false.
+	// This field is alpha-level and is honored only by servers that enable the PodShareProcessNamespace feature.
+	// +k8s:conversion-gen=false
```

Comment:
Why do you have conversion-gen=false on here?  Just because the predecessors have it?

Reply:
I thought it would be necessary because the `Spec.ShareProcessNamespace` vs `Spec.SecurityContext.ShareProcessNamespace` path difference needed manual conversion.